docker/candlepin-base/setup-supervisord.sh

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
Nah, that's fine. I thought we were only using supervisord for spinning up the databases; but if we're using it for other things, then we can just leave it. No reason to reinvent the wheel quite yet.